.github/workflows/test.yml

```diff
@@ -9,6 +9,20 @@ on:
       - '.gitignore'
 
 jobs:
+  bash-test:
+    runs-on: ubuntu-latest
```

Comment:
🔍 Ensure that 'tests/test_action_checks.sh' exists and is executable to avoid runtime errors.